js/forum/dist/extension.js

```diff
@@ -62,6 +63,27 @@ System.register("wuethrich44/sso/main", ["flarum/extend", "flarum/app", "flarum/
                     items.remove('changeEmail');
```

Comment:
If you remove the whole account section you can get rid of these function but if future releases add more options to the account section you're not safe 😕